_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -0,0 +1,599 @@
+						<path d="M689.118,75.821v-50.53c4.19,0,5.866-2.263,5.866-5.28h4.442v55.81H689.118z"/>
+						<path d="M781.464,35.765c-5.028-4.609-12.402-8.967-22.374-8.967c-14.916,0-23.296,10.225-23.296,20.867v1.089
+							c0,10.558,8.464,20.445,24.05,20.445c9.303,0,17.012-4.441,21.872-8.965L788,66.854C781.883,72.887,771.492,78,759.174,78
+							c-21.118,0-33.939-13.743-33.939-28.828v-1.843c0-15.084,13.993-29.329,34.44-29.329c11.816,0,22.541,4.944,28.324,11.146
+							L781.464,35.765z"/>
+						<path d="M299.82,37.417c1.889,1.218,2.418,3.749,1.192,5.648l-9.553,14.797c-1.226,1.901-3.752,2.452-5.637,1.234l0,0
+							c-1.886-1.22-2.421-3.745-1.192-5.647l9.553-14.797C295.41,36.753,297.935,36.201,299.82,37.417L299.82,37.417z"/>
+					</g>
+					</svg>
+				</div>
+				<div class="jetpack-about__content">
+					<div class="jetpack-about__images">
+						<ul class="jetpack-about__gravatars">
+							<?php $this->display_gravatars(); ?>
+						</ul>
+						<p class="meet-the-team">
+							<a href="https://automattic.com/about/" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_meet_the_team"><?php esc_html_e( 'Meet the Automattic team', 'jetpack' ); ?></a>
+						</p>
+					</div>
+
+					<div class="jetpack-about__text">
+						<p>
+							<?php esc_html_e( 'We are the people behind WordPress.com, WooCommerce, Jetpack, Simplenote, Longreads, VaultPress, Akismet, Gravatar, Crowdsignal, Cloudup, and more. We believe in making the web a better place.', 'jetpack' ); ?>
+							<a href="https://automattic.com/" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_learn_more">
+								<?php esc_html_e( 'Learn more about us.', 'jetpack' ); ?>
+							</a>
+						</p>
+						<p>
+							<?php esc_html_e( 'We’re a distributed company with 864 Automatticians in 68 countries speaking 84 different languages. Our common goal is to democratize publishing so that anyone with a story can tell it, regardless of income, gender, politics, language, or where they live in the world.', 'jetpack' ); ?>
+						</p>
+						<p>
+							<?php esc_html_e( 'We believe in Open Source and the vast majority of our work is available under the GPL.', 'jetpack' ); ?>
+						</p>
+						<p>
+							<?php
+								// Maybe use printf() because we'll want to escape the string but still allow for the link, so we can't use esc_html_e().
+								echo wp_kses(
+									__( 'We strive to live by the <a href="https://automattic.com/creed/" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_creed">Automattic Creed</a>.', 'jetpack' ),
+									array(
+										'a' => array(
+											'href'   => array(),
+											'class'  => array(),
+											'target' => array(),
+											'data-jptracks-name' => array(),
+										),
+									)
+								);
+							?>
+						</p>
+						<p>
+							<a href="https://automattic.com/jobs" target="_blank"  class="jptracks" data-jptracks-name="jetpack_about_work_with_us">
```

Comment:
Should we use the final URL for that page instead?

```suggestion
							<a href="https://automattic.com/work-with-us/" target="_blank"  class="jptracks" data-jptracks-name="jetpack_about_work_with_us">
```

Reply:
Done